options/options.go

```diff
@@ -5,6 +5,8 @@ type Options struct {
 	SampleRate int    `long:"samplerate" description:"Only send 1 / N log lines" default:"1"`
 	WriteKey   string `long:"writekey" description:"Honeycomb team write key"`
 	StateDir   string `long:"statedir" description:"Directory where ingest state is stored" default:"."`
+	HighAvail  bool   `long:"highavail" description:"Enable high availability ingestion using DynamoDB"`
+	BackFill   int    `long:"backfill" description:"The number of hours to increase backfill of log ingestion to with max of 168 hours (1 week)" default:"1"`
```

Comment:
We have both `BackFill` and `Backfill` as variable names in this codebase - let's choose one and use it everywhere.  Also I'd like to suggest `BackfillHrs` since it's just an `int` to indicate that it's in hours